options/locale/locale_en-US.ini

```diff
@@ -170,7 +170,7 @@ string.desc = Z - A
 
 [error]
 occurred = An error occurred
-report_message = If you are sure this is a Gitea bug, please search for issues on <a href="https://github.com/go-gitea/gitea/issues" target="_blank">GitHub</a> or open a new issue if necessary.
+report_message = If you believe that this is a Gitea bug, please search for issues on <a href="%[1]s" target="_blank">GitHub</a> or open a new issue if necessary.
```

Comment:
It seems strange that the URL comes from template string but the link text is hard-coded here.

The GitHub should always mean "https://github.com/go-gitea/gitea/issues", right?

I would suggest don't change it at the moment.

Reply:
I also made GitHub a parameter, should I also make `Gitea` a parameter while I'm at it? I presume that you do not provide support for soft-forks like Forgejo, so it could make sense.

Of course, I'm willing to undo that change completely if possible and discuss this in a separate PR - Causing tension is not my intention (oh hey, that rhymes!).

Reply:
Never mind, either is fine to me as long as it makes sense. Thank you.